lib/include/public/ILogManager.hpp

```diff
@@ -49,6 +49,9 @@ namespace MAT_NS_BEGIN
     {
        public:
         /// <summary>
+        /// WARNING: If you are using offline storage you need to call Flush before FlushAndTearDown.
+        /// https://github.com/microsoft/cpp_client_telemetry/blob/main/docs/use-after-free.md#flushandteardown
+        ///
```

Comment:
While you are here, can you change the below comment too as `FlushAndTearDown` doesn't necessarily flush to persistent storage? I think just "tear-down the telemetry logging system" should be enough.

Reply:
Do you want me to modify the Flush() comment or remove the "Flushes any pending telemetry events in memory to disk" part of this comment?